docs/reference/query-dsl/semantic-query.asciidoc

```diff
@@ -72,7 +72,10 @@ For example, if you have a document that covers varying topics:
 ------------------------------------------------------------
 POST my-index/_doc/lake_tahoe
 {
-  "inference_field": [ "Lake Tahoe is the largest alpine lake in North America", "When hiking in the area, please be on alert for bears" ]
+  "inference_field": [
+    "Lake Tahoe is the largest alpine lake in North America",
+    "When hiking in the area, please be on alert for bears"
+  ]
 }
 ------------------------------------------------------------
 // TEST[skip:TBD]
```

Comment:
This is a total drive-by nit on docs side here, and no immediate action required, but might be good to start giving more explanatory reasons for skipping docs tests. We're currently auditing docs snippet testing coverage so it would be helpful :)

Even something like `//TEST[skip:Uses ML]` would be more informative.